### PR TITLE
[12.0][FIX] pos_require_product_quantity

### PR DESCRIPTION
**Pull request by ivantodorovich**
- FIX include : https://github.com/OCA/pos/commit/8d35da8d1c044c6dc0ee0f4302b1f62ce492a958
- Also took the opportunity to improve the code a bit

ping @robinkeunen

### CODE REVIEW

File: pos_require_product_quantity/static/src/js/screens.js [hunk 1]
```diff
@@ -16,39 +16,28 @@ odoo.define('pos_require_product_quantity.screens', function (require) {
...
-                    for (var i = 0; i < orderlines.length; i++) {
-                        var line = orderlines.models[i];
-                        if (line.quantity === 0) {
-                            qty_unset_list.push(line);
-                        }
-                    }
-                    if (qty_unset_list.length > 0) {
+            if (self.pos.config.require_product_quantity) {
+                this.$('.pay').click(function () {
+                    var lines_without_qty = _.filter(
```
Comment (robinkeunen) [inline pos_require_product_quantity/static/src/js/screens.js:21]: I'm a js n00b, what is the value of `_` here?

**Review by robinkeunen — APPROVED** [review]
LGTM, thanks for fixing and cleaning up my code :-)

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: pos_require_product_quantity/static/src/js/screens.js (hunk 1 above)
Comment (ivantodorovich) [reply pos_require_product_quantity/static/src/js/screens.js:21]: Just to simplify the code by using [underscore](https://underscorejs.org/) utility methods ☺️

In this case it resulted in cleaner code than a `for .. loop`

Comment (robinkeunen) [reply pos_require_product_quantity/static/src/js/screens.js:21]: Ha ok, it's a library, I thought it was some kind of js magic.

Beautiful, _kill all the loops_!

**Review by cubells — APPROVED** [review]
LGTM too

**Comment by ivantodorovich** [comment]
@OCA/pos-maintainers can we merge this?

**Comment by legalsylvain** [comment]
of course. 

/ocabot merge minor

**Pull request merged**